pyttb/sptensor.py

```diff
@@ -828,16 +1168,48 @@ def logical_xor(
 
     def mask(self, W: sptensor) -> np.ndarray:
         """
-        Extract values as specified by a mask tensor
+        Extract :class:`pyttb.sptensor` values as specified by `W`, a
+        :class:`pyttb.sptensor` containing only non-zero values that are
+        ones (1). The values in the :class:`pyttb.sptensor` corresponding
+        to the indices of the ones (1) in `W` will be returned as a column
+        vector.
 
         Parameters
         ----------
         W:
-            Mask tensor
+            Mask tensor to apply to ktensor.
```

Comment:
```suggestion
            Mask tensor to apply to sparse tensor.
```

Reply:
Actually streamlined to just "Mask tensor." to align with brevity throughout.